pom.xml

```diff
@@ -1,50 +1,93 @@
-<project>
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>org.threadly</groupId>
-  <artifactId>auroraArc</artifactId>
-  <version>0.4</version>
-  <packaging>jar</packaging>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.threadly</groupId>
+	<artifactId>auroraArc</artifactId>
+	<version>0.5</version>
+	<packaging>jar</packaging>
 
-  <name>AuroraArc</name>
-  <description>A JDBC driver for handling secondary reads and advanced failover capabilities with AWS Aurora database clusters.</description>
-  <url>https://github.com/threadly/auroraArc</url>
+	<name>AuroraArc</name>
```

Comment:
Why are you changing the spacing of this file?

Reply:
Standard eclipse formatting : CTRL+SHIFT+F . It is the standard formatting for a pom.xml file. tons of files out there are formatted this way...